src/app/clusters/client-monitoring-server/client-monitoring-server.cpp

```diff
@@ -41,12 +82,14 @@ bool emberAfClientMonitoringClusterRegisterClientMonitoringCallback(
 /**
  * @brief Client Monitoring Cluster StayAwakeRequest Command callback (from client)
  */
-bool emberAfClientMonitoringClusterStayAwakeRequestCallback(app::CommandHandler * commandObj,
-                                                            const chip::app::ConcreteCommandPath & commandPath,
+bool emberAfClientMonitoringClusterStayAwakeRequestCallback(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
                                                             const Commands::StayAwakeRequest::DecodableType & commandData)
 {
     emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_UNSUPPORTED_COMMAND);
```

Comment:
Pre-existing, but please don't use `emberAfSendImmediateDefaultResponse`.  We're trying to remove that.  Send responses via the CommandHandler.

Reply:
switched to CommandHandler